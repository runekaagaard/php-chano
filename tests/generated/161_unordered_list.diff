001- <li>item 1
001+ Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/161_unordered_list.php on line 16
002- 	<ul>
003- 		<li>item 1.1</li>
004- 	</ul>
005- 	</li>