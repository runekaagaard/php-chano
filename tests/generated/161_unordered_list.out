Fatal error: Call to undefined method DtlIter::unordered_list() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/161_unordered_list.php on line 16